data.py

```diff
@@ -58,13 +59,153 @@ def jitter_pointcloud(pointcloud, sigma=0.01, clip=0.05):
     return pointcloud
 
 
+class Jitter:
+    def __init__(self, sigma=0.01, clip=0.05):
+        self._sigma = sigma
+        self._clip = clip
+
+    def __call__(self, pointcloud):
+        return jitter_pointcloud(pointcloud, self._sigma, self._clip)
+
+
+class RandomRemove:
+    """
+    Remove pointcloud randomly with an arbitrary probability.
+    """
+    def __init__(self, prob=0.2):
+        self._prob = prob
+
+    def __call__(self, pointcloud):
+        original_length = pointcloud.shape[0]
+        random_uniform = np.random.uniform(0, 1, pointcloud.shape[0])
+        masked_in = random_uniform > 0.2
+
+        masked_pointcloud = pointcloud[masked_in, :]
+
+        if len(masked_pointcloud) == 0:
+            return pointcloud
+
+        while masked_pointcloud.shape[0] < original_length:
+            masked_pointcloud = np.concatenate([masked_pointcloud, masked_pointcloud], axis=0)
+
+        masked_pointcloud = masked_pointcloud[:original_length, :]
+        return masked_pointcloud
+
+
+class RandomRemoveFourQuadrant:
```

Comment:
This is the class that does the 25 percent occlusion procedure.

@FranzEricSchneider  will be in charge of the experiment and the validations